forge-gui/res/cardsfolder/upcoming/walking_bulwark.txt

```diff
@@ -4,7 +4,7 @@ Types:Artifact Creature Golem
 PT:0/3
 K:Defender
 A:AB$ Pump | Cost$ 2 | ValidTgts$ Creature.withDefender | KW$ Haste | SorcerySpeed$ True | TgtPrompt$ Select target creature with defender | SubAbility$ DBEffect | SpellDescription$ Until end of turn, target creature with defender gains haste, can attack as though it didn't have defender, and assigns combat damage equal to its toughness rather than its power. Activate only as a sorcery.
-SVar:DBEffect:DB$ Effect | RememberObjects$ Targeted | StaticAbilities$ CanAttack,CombatDamageToughness
+SVar:DBEffect:DB$ Effect | RememberObjects$ Targeted | StaticAbilities$ CanAttack,CombatDamageToughness | ForgetOnMoved$ Battlefield
 SVar:CanAttack:Mode$ CanAttackDefender | Defined$ Remembered | Description$ This creature gains haste, can attack as though it didn't have defender, and assigns combat damage equal to its toughness rather than its power.
```

Comment:
Statics don't have Defined

Reply:
for this static, it's called `ValidCard`